qemu/tests/cfg/blockdev_stream_install.cfg

```diff
@@ -37,6 +37,7 @@
 
     tag_for_install_start = "Starting Login Service|Starting Update is Completed"
     tag_for_install_start += "|Mounted Temporary Directory"
+    tag_for_install_start += "|Anaconda NetworkManager configuration"
```

Comment:
this tag is different from others, I guessed you may hit the issue that 'NetworkManager autoconnections configuration for Anaconda installation environment" was not found during installation, then what about adding both?

Reply:
In fact, ppc don't support this tag, and the related tag is "NetworkManager auonda installation environment" in ppc, so, to make the same tag for both x86 and ppc, I re-set the tag to "Anaconda NetworkManager configuration" which are supported on both arch.

Reply:
After discussion with ngu, updated patch according to zhenchao's suggestion.

Reply:
make sense, LGTM